### PR TITLE
[BCP][11.0] [IMP] avoid cycle on request

### PR DESCRIPTION
**Pull request by nilshamerlinck**
Backport of 1282375

### CODE REVIEW

File: addons/web/controllers/main.py [hunk 1]
```diff
@@ -655,6 +654,7 @@ def post(self, path):
             base_url = request.httprequest.base_url
             query_string = request.httprequest.query_string
             client = Client(request.httprequest.app, BaseResponse)
+            client = Client(http.root, BaseResponse)
```
Comment (gurneyalex) [inline addons/web/controllers/main.py:656]: I think the previous line (656) needs to be removed.

Comment (nilshamerlinck) [reply addons/web/controllers/main.py:656]: oups, correct, thanks!

**Review by florentx — CHANGES_REQUESTED** [review]
Missing an occurence of `request.httprequest.app` in `addons/website/models/website.py` if I'm not mistaken.

**Comment by nilshamerlinck** [comment]
Hi @florentx you're right! actually 2 references, one in _/web/proxy/load_ route as well

**Review by florentx — APPROVED** [review]

**Review by yvaucher — APPROVED** [review]

**Review by gurneyalex — APPROVED** [review]

**Pull request merged**